### ISSUE TITLE
[Feature]: ensemble model

### ISSUE THREAD

**Issue by typoverflow** [typoverflow/UtilsRL#13]
### Prerequisites

- [X] I have tried updating UtilsRL to the newest version.
- [X] I have checked both open and closed issues but found nothing related to my request.

### UtilsRL verison when proposing this request

0.4.6

### What I am expecting

An nn module that gives ensemble outputs to implement several tricks in modern RL research, e.g. ensemble dynamics models and ensemble Q networks.

### Urgency

Urgent, will bring significant improvement and should be considered in next main version.

**Comment by typoverflow** [comment]
The current implementation is not correct. Reverting.
```python
class EnsembleLinear(nn.Module):
    def __init__(
        self,
        in_features, 
        out_features, 
        ensemble_size: int = 1,
        bias: bool = True, 
        device: Optional[Any] = None, 
        dtype: Optional[Any] = None
    ):
        self.in_features = in_features
        self.out_features = out_features
        self.ensemble_size = ensemble_size
        self.linear = nn.Linear(in_features*ensemble_size, out_features*ensemble_size, bias, device, dtype)
        
    def forward(self, input: torch.Tensor):
        if input.shape[0] != self.ensemble_size:
            input = input.unsqueeze(0).repeat(self.ensemble_size)
        out = self.linear(input.permute(1, 0, 2).reshape(-1, self.in_features*self.ensemble_size))
        return out.reshape(-1, self.ensemble_size, self.out_features).permute(1, 0, 2)
```
The linear layer should not be densely connected.

**Comment by typoverflow** [comment]
I think this issue may be closed by #14.
